feat: implemented to_dense() to paddle frontend

Mac16661 opened the pull request:
Closes #26823 

- [x] Did you add a function?
- [x] Did you add the tests?
- [ ] Did you run your tests and are your tests passing?
- [x] Did pre-commit not fail on any check?
- [x] Did you follow the steps we provided?

juliagsy commented on ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py:
```diff
@@ -151,29 +151,6 @@ def _get_dtype_and_square_matrix(draw):
 
 
-# bmm helper function
-@st.composite
```
hey! thanks for contributing! please don't remove anything from the file. you should only add the test for the function you're working on, thanks!

Mac16661 replied:
I made a git pull request . I think that's why  some line of code got deleted

Mac16661 commented:
Actually I didn't  removed any thing. I have just added the to_dense test
function. May be I did git pull that's why some code from the file is
removed.
On Fri, 13 Oct, 2023, 6:42 pm JG, ***@***.***> wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In
> ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py
> <https://github.com/unifyai/ivy/pull/26915#discussion_r1358248967>:
>
> > @@ -151,29 +151,6 @@ def _get_dtype_and_square_matrix(draw):
>      return dtype, mat
>
>
> -# bmm helper function
> ***@***.***
>
> hey! thanks for contributing! please don't remove anything from the file.
> you should only add the test for the function you're working on, thanks!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/unifyai/ivy/pull/26915#pullrequestreview-1676477971>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AO7KXV56F2VYASPYZIMVT7DX7E44NANCNFSM6AAAAAA54KFFVU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

juliagsy commented:
ahh but the PR diff is showing the removes, and they will be pushed altogether if we were to merge, could you please make sure that your fork+branch are up to date with the latest commit on master (then include your frontend function changes) and either update the PR again or open a new one if the latter simplifies things? Thanks!

Mac16661 commented:
feat(paddle_frontend): created a new pull request for to_dense method and test function #27040
